License2Deploy/rolling_deploy.py

```diff
@@ -400,13 +403,16 @@ def get_args(): # pragma: no cover
   parser.add_argument('-r', '--ready-wait', action='store', dest='ready_wait', help='Wait time for ec2 instance to be ready', type=int, nargs=2, default=[10, 30])
   parser.add_argument('-H', '--health-wait', action='store', dest='health_wait', help='Wait time for ec2 instance health check', type=int, nargs=2, default=[10, 30])
   parser.add_argument('-o', '--only-new-wait', action='store', dest='only_new_wait', help='Wait time for old ec2 instances to terminate', type=int, nargs=2, default=[10, 30])
+  parser.add_argument('-A', '--asg-logical-name', action='store', dest='asg_logical_name', help='ASG Logical Name from CFN', type=str)
```

Comment:
hmm. I don't like these 1-letter arguments. especially with -a already being taken by --ami.

As I understand it, these are not used. I would say use a different letter at least if you want to keep in the 1-letter args.

Reply:
will approve if you think there is a good reason to leave as is

Reply:
These are valid concerns. Single character arguments are pretty standard. Not sure what you mean by 'not used', these are definitely used, the command from deploy_ami job is `/usr/bin/rolling_deploy -f ${FORCE_REDEPLOY} ${DEPLOY_OPTIONS} -e qa -p cos-service -b ${BUILD_NUM} -P dnbi-deploy -a ${AMI} -s ${stack_name}`. 

Do you have a recommendation for what to use instead of -A? I ask b/c we use -c/-C and -p/-P already in this script and I chose -A since ASG is capitalized in the logical name so it made sense. I don't mind changing if you have a suggestion for something that would make more sense.

Reply:
Oh wow. I assumed the deploy_ami job would have used the longer args so you could understand the difference between -p/-P. Especially since they are in a jenkins job and not something that is being typed repeatedly. I vote for updating that job to use longer args but will approve this now as it is keeping with the existing convention.